pages/admin/invite/userAccountPage/[id].tsx

```diff
@@ -69,7 +312,7 @@ const UserAccountPage: React.FunctionComponent<UserRequest> = ({
       if (!response.ok) {
         throw await response.json();
```

Comment:
When trying to accept a user an error is thrown: "Error: "status" is not allowed". I think it might be the type that you are passing in as "unknown". You will need to also add `status` to the expected body of the API endpoint that you use.

Reply:
Make sure that after you fix the accept button here that you also update the accept button at ./components/userRequest as well!!